build/Build.Pack.cs

```diff
@@ -193,22 +193,38 @@ void CreateLinuxPackages(string runtimeId)
 
             foreach (var runtimeId in RuntimeIds.Where(x => x.StartsWith("win")))
             {
-                var framework = runtimeId.Equals("win") ? NetFramework : NetCore;
+                switch (runtimeId)
+                {
+                    case "win":
+                        PackWindowsZip(NetFramework, runtimeId);
+                        break;
+                    case "win-x86":
+                    case "win-x64":
+                        PackWindowsZip(NetCoreWindows, runtimeId);
+                        PackWindowsZip(NetCore, runtimeId);
```

Comment:
On the surface it is a little confusing, do we end up packing the net core for windows stuff twice? But we don't below for arm?

I wonder why we do that?